### PR TITLE
Optimize Profile and Home page context

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
#### What are the relevant tickets?
Fixes: #1016 

#### What's this PR do?
Optimize Profile and Home page context

#### How should this be manually tested?
Navigate to Home and Product pages and check if the sequence of subsections is correct, along with its data.

### CODE REVIEW

**Comment by ahmed-belal** [comment]
@HamzaIbnFarooq It would be better to include "verify that there are no duplicate queries on the home and product page rendering" as that is the original issue being fixed.

**Review by ahmed-belal — APPROVED** [review]
LGTM

**Comment by HamzaIbnFarooq** [comment]
@ahmed-belal, After profiling Home and Product pages, with current PR's fixes implemented, I could only find some image related duplicate queries which seems like a technical limitation. Other than that everything seems in-place.

File: cms/models.py [hunk 1]
```diff
@@ -127,9 +127,14 @@ class HomePage(Page, CommonProperties):
     def get_context(self, request, *args, **kwargs):
         return {
             **super().get_context(request, *args, **kwargs),
+            "alumni": self.alumni,
+            "catalog": self.catalog,
+            "CSOURCE_PAYLOAD": None,
+            "learning_resources": self.learning_resources,
+            "program_description_section": self.program_description_section,
             "site_name": settings.SITE_NAME,
+            "three_column_image_text_section": self.three_column_image_text_section,
```
Comment (gsidebo) [inline cms/models.py:136]: One very minor suggestion: can you put the variables that are not properties of this class above the other variables that are properties?
```suggestion
            "site_name": settings.SITE_NAME,
            "CSOURCE_PAYLOAD": None,
            "alumni": self.alumni,
            "catalog": self.catalog,
            "learning_resources": self.learning_resources,
            "program_description_section": self.program_description_section,
            "three_column_image_text_section": self.three_column_image_text_section,
```

**Review by gsidebo — CHANGES_REQUESTED** [review]

File: cms/models.py (hunk 1 above)
Comment (HamzaIbnFarooq) [reply cms/models.py]: @gsidebo, Thanks for the review.
I have reordered the variables along with adding a comment, kindly have a look now.

**Review by gsidebo — APPROVED** [review]
Looks great! 👍

**Pull request merged**